{"name": "Проверка авторизации пользователя с невалидными данными", "status": "passed", "parameters": [{"name": "user_data", "value": "{'email': <bound method Provider.free_email of <faker.providers.internet.en_US.Provider object at 0x000001E55B1B7AD0>>, 'password': '7862535'}"}], "start": 1727866297706, "stop": 1727866298262, "uuid": "d3f0f55d-7023-41ad-8c32-7d5f9eb043cb", "historyId": "2bedd92e86293a4b207157a0c862e1b3", "testCaseId": "1bdf4b355a3b6f766823fc9d5e792d25", "fullName": "tests.test_login_user.TestLoginUser#test_login_with_incorrect_data_success", "labels": [{"name": "suite", "value": "Тестирование логина пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "User-PC"}, {"name": "thread", "value": "34856-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}